lib/src/commands.dart

```diff
@@ -452,6 +476,35 @@ class CommandsPlugin extends BasePlugin implements ICommandGroup<IContext> {
     );
   }
 
+  Future<AutocompleteContext> _autocompleteContext(
+    IAutocompleteInteractionEvent interactionEvent,
+    ChatCommand command,
+  ) async {
+    ISlashCommandInteraction interaction = interactionEvent.interaction;
```

Comment:
why explicit type over `final`?

Reply:
you got IDE to tell you what variable contains also now you variable is mutable which is not good and that style is not consistent with official dart guides. But if you got that situation in whole project its better to keep it that way. I would consider switching to final on other occasion